docker-images/Dockerfile.oni-runtime-debug

```diff
@@ -0,0 +1,28 @@
+ARG GO_VERSION=1.14.2
```

Comment:
Now that we have more images, can we turn the `HISTORY.md` into a `README.md` (moving all its content to a section `## Image version history`, and describing in one line what each of the three images contains/does?